docker/Dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM bitnami/minideb:stretch
+FROM bitnami/minideb:bullseye
```

Comment:
bullseye installs java11. Otherwise we might need to mess with the sources.
Another alternative is using sid which has openjdk-11-jdk available as a package.

Reply:
yep, bullseye gets java11 which is suitable for hail.